ocs_ci/helpers/helpers.py

```diff
@@ -716,6 +716,76 @@ def create_multiple_pvcs(
     return ocs_objs
 
 
+def create_bulk_pvcs(
+    sc_name,
+    namespace,
+    number_of_pvc=1,
+    size=None,
+    access_mode=constants.ACCESS_MODE_RWO,
+):
+    """
+    Create one or more PVC as a bulk or one by one
+    Args:
+        sc_name (str): The name of the storage class to provision the PVCs from
+        namespace (str): The namespace for the PVCs creation
+        number_of_pvc (int): Number of PVCs to be created
+        size (str): The size of the PVCs to create
+        access_mode (str): The kind of access mode for PVC
+
+    Returns:
+         list: List of PVC objects and tmp dir where the yamls are created
+    """
+
+    pvc_data = templating.load_yaml(constants.CSI_PVC_YAML)
+    pvc_data["metadata"]["namespace"] = namespace
+    pvc_data["spec"]["accessModes"] = [access_mode]
+    pvc_data["spec"]["storageClassName"] = sc_name
+    if size:
+        pvc_data["spec"]["resources"]["requests"]["storage"] = size
+    if access_mode == "ReadWriteMany" and "rbd" in sc_name:
+        pvc_data["spec"]["volumeMode"] = "Block"
+    else:
+        pvc_data["spec"]["volumeMode"] = None
+
+    # Creating tem directory to hold the files for the PVC creation
+    tmpdir = tempfile.mkdtemp()
+    logger.info("Creating the PVC yaml files for creation in bulk")
+    ocs_objs = []
+    for _ in range(number_of_pvc):
+        name = create_unique_resource_name("test", "pvc")
+        logger.info(f"Adding PVC with name {name}")
+        pvc_data["metadata"]["name"] = name
+        templating.dump_data_to_temp_yaml(pvc_data, f"{tmpdir}/{name}.yaml")
+        ocs_objs.append(pvc.PVC(**pvc_data))
+
+    logger.info("Creating all PVCs as bulk")
+    oc = OCP(kind="pod", namespace=defaults.ROOK_CLUSTER_NAMESPACE)
```

Comment:
This function is in a way duplicate of create_multiple_pvcs and following Avi's review, I've removed it. In the create_multiple_pvcs function, the correct namespace is used

Reply:
@keesturam  - I think that this validation is more accutate than comparind all the images to all the PVs, because there might be leftovers from the previous test runs. 
And check if the total rbd images in ceph system matchs with the PV count at OCS should be part of the improved flow of all the tests ( to measure images and pv count before the test and if not equal  - exit). 
But not an intergral part of this specific test.